passdmenu.py

```diff
@@ -264,14 +285,17 @@ def main():
     if choice is None:
         sys.exit(0)
     pass_output = get_pass_output(choice, args.pass_bin, args.store)
-    user, pw = get_user_pw(pass_output)
+    user, pw = get_user_pw(pass_output, args.get_user)
 
     info = []
-    if args.get_user and user is not None:
+    if user is not None:
         info += [user]
     if args.get_pass and pw is not None:
         info += [pw]
 
+    if len(info) == 0:
+        info = [' ']
+
```

Comment:
I was convinced that passing a completely empty bytes object to the xclip subprocess causes problems. Rechecking made me doubt this observation :) I'll remove those lines, they're indeed unnecessary.